composer.json

```diff
@@ -1,8 +1,27 @@
 {
+    "name": "automattic/tumblr-importer",
+    "description": "Tumblr Importer is a plugin that imports posts from Tumblr into WordPress.",
+    "license": "GPL-2.0",
+    "type": "wordpress-plugin",
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.a8c.com:Automattic/redvelvet-lib.git"
```

Comment:
using vcs for now, need to upload to packagist

Reply:
we will not use this for now, we will use this library in the future. we could even remove it now, as the purpose of this PR is to use composer/autoloader